zhaquirks/philips/__init__.py

```diff
@@ -288,15 +294,21 @@ def send_press_event(click_count):
                 sim_event_args[ARGS][2] = 2
                 action = f"{button.action}_{press_type.action}"
                 _LOGGER.debug(
-                    "%s - send_press_event emitting simulated action: [%s]",
+                    "%s - send_press_event emitting simulated action: [%s] event_args: %s",
```

Comment:
Adding a comma to the log message here to be similar to others:
```suggestion
                    "%s - send_press_event emitting simulated action: [%s], event_args: %s",
```